go/felix/set/set.go

```diff
@@ -56,6 +70,14 @@ func (set mapSet) Add(item interface{}) {
 	set[item] = emptyValue
 }
 
+func (set mapSet) AddAll(itemArray interface{}) {
+
+	arrVal := reflect.ValueOf(itemArray)
```

Comment:
Interesting.  Go's type checking is normally compile time, isn't it?  But here I guess it must be at runtime instead?